add general note component, add notes to classic guides to clarify OIE behavior (#3014)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/customize-themes/index.md

**What you need**

* [Okta Developer Edition organization](https://developer.okta.com/signup)\
* [Postman](https://www.postman.com/downloads/)

---

## About theme customization of your Okta org

> Okta provides you with a lot of power to authenticate your users, and using the [Okta-hosted Sign-In Widget](/docs/concepts/redirect-vs-embedded/#redirect-authentication) gets you up and running quickly, without having to write much custom code or host the sign-in functionality yourself. However, the trade-off is that your [customization options](/docs/guides/custom-widget/) are more limited, and potentially tricky to administer (with the existing functionality you tend to need to set colors, icons, and so on, in multiple places).

Reviewer: ```suggestion
Okta provides you with a lot of power to authenticate your users, and using the [Okta-hosted Sign-In Widget](/docs/concepts/redirect-vs-embedded/#redirect-authentication) gets you up and running quickly without having to write much custom code or host the sign-in functionality yourself. However, the trade-off is that your [customization options](/docs/guides/custom-widget/) are more limited and potentially tricky to administer, since the existing functionality that you typically need to set are colors, icons, and so on in multiple places.
```

[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/1d58ab8a3909dd6a3cfb)

> > **Note**: To make branding updates using the admin interface, see [Branding](https://help.okta.com/en/prod/Content/Topics/Settings/branding.htm?cshid=csh-branding).

Reviewer: ```suggestion
> **Note**: To make branding updates using the Admin Console, see [Branding](https://help.okta.com/en/prod/Content/Topics/Settings/branding.htm?cshid=csh-branding).
```

## Important: Overriding themes in the code template editors

> You can customize individual parts of your Okta org experience using the various code editors we've made available in the Okta Admin Console (for example, **Customizations** > **Sign-in page code editor** to customize the Okta-hosted sign-in page). Note that when you use the Brands API, your custom brand settings won't apply to the places where you've customized the theme settings.

Reviewer: ```suggestion
You can customize individual parts of your Okta org experience using the various code editors we've made available in the Admin Console (for example, **Customizations** > **Sign-in page code editor** to customize the Okta-hosted sign-in page). Note that when you use the Brands API, your custom brand settings won't apply to the places where you've customized the theme settings.
```
